src/main/java/com/thindeck/steps/Nginx.java

```diff
@@ -136,7 +136,9 @@ public void update(@NotNull final String host, @NotNull final int hport,
      * @param sport Server port to redirect requests to
      * @return Commands for updating hosts configuration.
      */
-    private String updateConfigScript(final String host, final String server,
+    private String updateLoadBalancingScript(
+        final String host,
```

Comment:
If the params fit in line they should be in the same as the method name (80 chars per line)